src/frontend/templates/components/Filters.js

```diff
@@ -71,12 +71,12 @@ export const Filters = () => {
                 <div className="px-xs pt-xs">
                     <label className='d-flex px-xs ai-c gap-1'> 
                     <input onChange={() =>  filtersDispatcher({ type: 'sort', payload: { ...filtersState.sort , sortBy: 'highlow' } })}  
-                    id='highlow' type='radio' name='sort' /> 
+                    id='highlow' type='radio' name='sort' checked={filtersState.sort.sortBy === 'highlow'} /> 
```

Comment:
Is the id required here, remove if not

Reply:
> Is the id required here? Remove if not
No Om , at this moment , they're not required .